Code/Tools/TestImpactFramework/Runtime/Native/Code/Source/TestRunner/Native/Job/TestImpactNativeTestJobInfoUtils.h

```diff
@@ -17,4 +22,24 @@ namespace TestImpact
     //! Generates the command string to launch the specified test target.
     AZStd::string GenerateLaunchArgument(
         const NativeTestTarget* testTarget, const RepoPath& targetBinaryDir, const RepoPath& testRunnerBinary);
+
+    //! Generates a test enumeration job command.
+    typename NativeTestEnumerator::Command GenerateTestEnumeratorJobInfoCommand(
+        const AZStd::string& launchArguement, const RepoPath& runArtifact);
+
+    //! Generates a regular test run job command.
+    typename NativeRegularTestRunner::Command GenerateRegularTestJobInfoCommand(
```

Comment:
repeat: typename is not needed here.
```suggestion
    NativeTestEnumerator::Command GenerateTestEnumeratorJobInfoCommand(
        const AZStd::string& launchArguement, const RepoPath& runArtifact);

    //! Generates a regular test run job command.
    NativeRegularTestRunner::Command GenerateRegularTestJobInfoCommand(
```